doc/classes/XRInterface.xml

```diff
@@ -122,17 +122,17 @@
 				[param mode] is the environment blend mode starting with the next frame.
 				[b]Note:[/b] Not all runtimes support all environment blend modes, so it is important to check this at startup. For example:
 				[codeblock]
-				                func _ready():
-				                    var xr_interface: XRInterface = XRServer.find_interface("OpenXR")
-				                    if xr_interface and xr_interface.is_initialized():
-				                        var vp: Viewport = get_viewport()
-				                        vp.use_xr = true
-				                        var acceptable_modes = [ XRInterface.XR_ENV_BLEND_MODE_OPAQUE, XRInterface.XR_ENV_BLEND_MODE_ADDITIVE ]
-				                        var modes = xr_interface.get_supported_environment_blend_modes()
-				                        for mode in acceptable_modes:
-				                            if mode in modes:
-				                                xr_interface.set_environment_blend_mode(mode)
-				                                break
+				func _ready():
+					var xr_interface: XRInterface = XRServer.find_interface("OpenXR")
+					if xr_interface and xr_interface.is_initialized():
+						var vp: Viewport = get_viewport()
+						vp.use_xr = true
+						var acceptable_modes = [XRInterface.XR_ENV_BLEND_MODE_OPAQUE, XRInterface.XR_ENV_BLEND_MODE_ADDITIVE]
+						var modes = xr_interface.get_supported_environment_blend_modes()
+						for mode in acceptable_modes:
+							if mode in modes:
+								xr_interface.set_environment_blend_mode(mode)
+								break
```

Comment:
```suggestion
				    var xr_interface: XRInterface = XRServer.find_interface("OpenXR")
				    if xr_interface and xr_interface.is_initialized():
				        var vp: Viewport = get_viewport()
				        vp.use_xr = true
				        var acceptable_modes = [XRInterface.XR_ENV_BLEND_MODE_OPAQUE, XRInterface.XR_ENV_BLEND_MODE_ADDITIVE]
				        var modes = xr_interface.get_supported_environment_blend_modes()
				        for mode in acceptable_modes:
				            if mode in modes:
				                xr_interface.set_environment_blend_mode(mode)
				                break
```
Spaces for indentation

Reply:
To be clearer, 4 spaces are used for indentation across all of the class reference documentation, as tabs are removed when the description is generated. The static check will fail because of this.

Reply:
It however does not fail in a clear way, hence the suggestion :)